content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -259,6 +259,93 @@ public class Application {
 }
 ```
 
+##### Configure the exporter programmatically
+
+You can also configure OTLP exporters programmatically. This configuration will
+replace the default OTLP exporter, and will add a custom header to the requests.
+
+```java
+package io.opentelemetry.example.graal;
+
+import static io.opentelemetry.exporter.otlp.internal.OtlpConfigUtil.DATA_TYPE_TRACES;
+
+import io.opentelemetry.api.metrics.MeterProvider;
+import io.opentelemetry.exporter.otlp.http.trace.OtlpHttpSpanExporter;
+import io.opentelemetry.exporter.otlp.http.trace.OtlpHttpSpanExporterBuilder;
+import io.opentelemetry.exporter.otlp.internal.OtlpConfigUtil;
```

Comment:
is there a way to avoid using internal classes?

Reply:
Only if we implement https://github.com/open-telemetry/opentelemetry-java/pull/6368

Reply:
BTW, the same class is added in https://github.com/open-telemetry/opentelemetry-java-examples/pull/378

I'm planning to use an "include" mechanism: https://github.com/open-telemetry/opentelemetry.io/issues/4276

Reply:
> is there a way to avoid using internal classes?

done :smile: